src/components/PupilViews/PupilQuiz/PupilQuiz.view.tsx

```diff
@@ -69,6 +74,28 @@ const QuizInner = () => {
   const isPartiallyCorrect = currentQuestionState?.isPartiallyCorrect;
   const isCorrect = grade === 1;
 
+  const handleNextQuestionClick = () => {
+    const _currentQuestionIndex = Math.min(
+      currentQuestionIndex + 1,
+      numQuestions,
+    );
+    if (_currentQuestionIndex === numQuestions) {
+      completeActivity(currentSection);
+      if (currentSection === "starter-quiz") {
+        track.lessonActivityCompletedStarterQuiz(
+          getQuizTrackingData(currentSection),
+        );
+      }
+      if (currentSection === "exit-quiz") {
+        track.lessonActivityCompletedExitQuiz(
+          getQuizTrackingData(currentSection),
+        );
+      }
+    } else {
+      handleNextQuestion();
```

Comment:
Are we tracking individual question results yet ? Could here be the place to do this ?

Reply:
no not yet; but i guess this could definitely be the place if/when we do